Dockerfile

```diff
@@ -13,6 +13,9 @@ RUN mkdir -p /OpenROAD-flow
 
 COPY --from=openroad /OpenROAD/build ./tools/build/OpenROAD
 COPY --from=openroad/yosys /build ./tools/build/yosys
+COPY --from=openroad/lsoracle /LSOracle ./tools/build/LSOracle
+COPY --from=openroad/lsoracle /LSOracle/core/test.ini /usr/local/share/lsoracle/
```

Comment:
I guess this will work in a container but why do we need to install in /usr/local?  That won't be possible in a bare metal build.

Reply:
Just following the conventions for linux filesystem.